src/engine/source/hlp/CMakeLists.txt

```diff
@@ -48,3 +49,13 @@ target_include_directories(hlp PUBLIC
     ${RapidJSON_SOURCE_DIR}/include/
     ${ENGINE_HLP_INCLUDE_DIR}
 )
+
+####################################################################################################
+# Test tool
+####################################################################################################
+
+add_executable(hlp_cli
+  ${ENGINE_HLP_TOOLS_DIR}/hlp_cli.cpp
+)
+
+target_link_libraries(hlp_cli hlp ${EXTERNAL_DEPS})
```

Comment:
add empty newline at the end of the file

Reply:
Fixed